packages/api-client/src/granules.ts

```diff
@@ -115,6 +115,7 @@ export const waitForGranule = async (params: {
 
   await pRetry(
     async () => {
+      // TODO update to use collectionId + granuleId
```

Comment:
This will need to be done in https://bugs.earthdata.nasa.gov/browse/CUMULUS-3015